File: game/Assets/Scripts/Network/Messages/ClientException.cs
// Extension of BugReport for uploading exceptions from the client.
using System;
using System.Collections.Generic;

namespace Network
{
    [Serializable]
    public class ClientException
    {
        public BugReport bug_report;
        public string condition;
        public string stack_trace;
        public string game_id;
        public string role;
    }
}

File: server/messages/client_exception.py
"""Extension of BugReport for uploading exceptions from the client."""

from dataclasses import dataclass

from mashumaro.mixins.json import DataClassJSONMixin

from server.messages.bug_report import BugReport


@dataclass(frozen=True)
class ClientException(DataClassJSONMixin):
    bug_report: BugReport
    condition: str
    stack_trace: str
    game_id: str
    role: str
    type: str
